README.md

```diff
@@ -1,2 +1,101 @@
 # Soneta.Platform.Developer
-Soneta.Platform.Developer
+[![NuGet](https://img.shields.io/nuget/v/Soneta.Platform.Developer.svg)](https://www.nuget.org/packages/Soneta.Platform.Developer/)
+[![NuGet](https://img.shields.io/nuget/dt/Soneta.Platform.Developer.svg)](https://www.nuget.org/packages/Soneta.Platform.Developer/)
+[![Build Status](https://dev.azure.com/soneta/GitHub/_apis/build/status/Soneta.Platform.Developer%20CI?branchName=master)](https://dev.azure.com/soneta/GitHub/_build/latest?definitionId=3&branchName=master)
+
+# Wstęp
+
+Soneta Platform Devlper jest to dodatek zawierający zestaw szablonów implementujący [Soneta.sdk](https://github.com/soneta/Soneta.MsBuild.SDK). Soneta Platform Devlper zachęca programistę do wprowadzenia modułowości w swoich dodatkach, poprzez rozdzielenie logiki biznesowaje, interfejsu użytkownika oraz testów od siebie. Jest to realizowane dzięki automatycznemu utworzeniu 3 typów projeków, które odpowiadają wyżej wspomnianemu podziałowi. W celu zachowania pełnej elastyczności i swobody w projektowaniu programista ma możliwość utworzenia każdego typu projektu osobno. <br>
```

Comment:
może coś takiego 
Soneta Platform Developer jest to rozszerzenie Visual Studio 2019 zawierające zestaw szablonów Visual Studio, które implementują elementy Soneta SDK. Soneta Platform Developer promuje rozdzielanie logiczne tworzonych solucji rozszerzeń zgodnie z architekturą enova365, poprzez podział rozwiązania na osobne projekty warstwy logiki biznesowej, warstwy interfejsu użytkownika oraz warstwy testującej. Realizacja takiego uporządkowania polega na automatycznym utworzeniu przez SDK trzech projektów według typów, które odpowiadają wyżej wspomnianemu podziałowi. W celu zachowania pełnej elastyczności i swobody w projektowaniu twórca rozszerzenia enova365 ma możliwość utworzenia każdego typu projektu oddzielnie.